core/src/main/java/org/apache/accumulo/core/file/rfile/CreateEmpty.java

```diff
@@ -40,7 +41,7 @@
  * Create an empty RFile for use in recovering from data loss where Accumulo still refers internally
  * to a path.
  */
-public class CreateEmpty {
+public class CreateEmpty implements KeywordExecutable {
```

Comment:
```suggestion
@AutoService(KeywordExecutable.class)
public class CreateEmpty implements KeywordExecutable {
```
Should this be added here?

Reply:
Yep, thanks. I forgot the annotation. I updated both PRs.